<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Upload New User Data to Cloud Demo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload User</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>62</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fName">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lName">
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="age">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>135</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="inputF"/>
     </item>
     <item>
      <widget class="QLineEdit" name="inputL"/>
     </item>
     <item>
      <widget class="QLineEdit" name="inputA"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="attempt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
